### ISSUE TITLE
Installer password confirm mismatch

### ISSUE THREAD

**Issue by Kexkey** [SatoshiPortal/cyphernode#102]
If password doesn't match on confirm, I can't go back and just retype the first time

**Comment by schulterklopfer** [comment]
yes. unfortunately with this type of installer, there is no way back. It needs some thinking how to implement "grouped" questions which will jump back to the first question in the "group" in case anything goes wrong in one of the questions inside the "group".

**Comment by Kexkey** [comment]
Workaround: ctrl-c and restart setup.
